Sources/Apollo/ApolloClient.swift

```diff
@@ -154,6 +153,124 @@ extension ApolloClient: ApolloClientProtocol {
                                       callbackQueue: queue,
                                       completionHandler: resultHandler)
   }
+
+#if swift(>=5.5.2) && canImport(_Concurrency)
+  @available(macOS 10.15, iOS 13, watchOS 8, tvOS 15, *)
```

Comment:
TODO: Check what the minimum available versions are for watchOS and tvOS. Does this library care about linux support?